cmd/goacme/main_test.go

```diff
@@ -22,15 +22,15 @@ func TestDiscoAliasFlag(t *testing.T) {
 		{defaultDisco, []string{"-d", "https://disco"}, "https://disco"},
 	}
 	for i, test := range tests {
-		var a discoAliasFlag = test.a
+		var daf = test.a
```

Comment:
it's not a rule set in stone. let me rephrase it. you're trying to fix a
golint suggestion. why make absolutely unrelated change, i.e. rename a
variable?
On 10 Jun 2016 6:20 p.m., "Kevin Kirsche" <notifications@github.com> wrote:

> In cmd/goacme/main_test.go
> <https://github.com/google/goacme/pull/29#discussion_r66649586>:
>
> > @@ -22,15 +22,15 @@ func TestDiscoAliasFlag(t *testing.T) {
> >  		{defaultDisco, []string{"-d", "https://disco"}, "https://disco"},
> >  	}
> >  	for i, test := range tests {
> > -		var a discoAliasFlag = test.a
> > +		var daf = test.a
>
> I can, any reason though? Just want to understand the reasoning. Go from
> how I've understood things usually follows the principle of using the first
> letter from each major word in the represented type / variable. In this
> case discoAliasFlag becomes daf.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/google/goacme/pull/29/files/c9559aaf24d623342dae02a86a7d630ea6d18668#r66649586>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AABjPeE7wFbHpxp8wq07R1e9olkGx2y8ks5qKZzugaJpZM4IzHtK>
> .
>